### DISCUSSION TITLE
小爱音箱播放文本以及执行文本指令功能

### DISCUSSION THREAD

**Discussion by oopuuu** [XiaoMi/ha_xiaomi_home#4] [Ideas]
试了一下目前的插件（v0.1.0），小爱触屏音箱第一代，似乎不支持播放指定的文本，也不支持执行指定的文本指令功能。来这里许个愿希望能后续官方能实现，尤其是播放文本，这个挺实用的！之前用miiotauto 有这个功能
<img width="374" alt="image" src="https://github.com/user-attachments/assets/504c470a-aba5-4083-a8c3-3e5b36d5d15a" />

**Comment by oopuuu** [comment]
这是miiotauto插件下的小爱触屏音箱的功能
<img width="671" alt="image" src="https://github.com/user-attachments/assets/7a9398d8-006b-429c-822c-c7e20f7c9da2" />

**Comment by oopuuu** [comment]
另外，发现小爱音箱有两个通知程序，和我说的两个需求有些相似，但似乎无法调用，也不知道这两个通知程序是做什么用的
<img width="1096" alt="image" src="https://github.com/user-attachments/assets/9ad9a4cd-41db-4748-b204-aa171feebfd8" />

**Comment by topsworld** [comment]
您好，集成是支持这些的，可以打开集成的【Action调试模式】，然后会在设备中枢出现一个调试用文本框
![image](https://github.com/user-attachments/assets/b9686523-5c5d-4d4c-8d37-16b182f33f82)
然后按照指定格式输入内容就可以，触发格式可以打开通知程序中的属性查看
![image](https://github.com/user-attachments/assets/34ccb717-cc5c-48ca-90f1-a2b3bc23ca3d)

![image](https://github.com/user-attachments/assets/195dd862-1957-4814-a4d2-1823833872a6)

**Reply by topsworld** [reply]
也可以直接自动化中调用这个通知程序实体，按照格式输入触发命令就可以

**Reply by oopuuu** [reply]
> 也可以直接自动化中调用这个通知程序实体，按照格式输入触发命令就可以

太好了，谢谢

**Comment by oopuuu** [comment]
已解决，需要开启action调试模式

**Reply by topsworld** [reply]
这个调试模式展示的文本框一般用于调试，调试没问题以后可以调用通知程序中的实体做触发

**Reply by oopuuu** [reply]
> 这个调试模式展示的文本框一般用于调试，调试没问题以后可以调用通知程序中的实体做触发

<img width="734" alt="image" src="https://github.com/user-attachments/assets/303ddd56-1630-4de9-931c-c76f8428d1cf" />
提示这个大概是啥毛病呢

**Reply by oopuuu** [reply]
> > 这个调试模式展示的文本框一般用于调试，调试没问题以后可以调用通知程序中的实体做触发
> 
> <img alt="image" width="734" src="https://private-user-images.githubusercontent.com/75833466/396099516-303ddd56-1630-4de9-931c-c76f8428d1cf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzNTMzODQsIm5iZiI6MTczNDM1MzA4NCwicGF0aCI6Ii83NTgzMzQ2Ni8zOTYwOTk1MTYtMzAzZGRkNTYtMTYzMC00ZGU5LTkzMWMtYzc2Zjg0MjhkMWNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjE2VDEyNDQ0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWExN2FmODgxYTEwNTk2ZTgyNGRhZjBhZjRlMjI2MGYwNWE4YWM5ZjdjZTY0Yjg4ZDM0ZTNiYzk1ZGVlMThmZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.b6Rm_l_4y7lONY9tg-pqyZbgYYqcZ2HkmnHcBkVxQ24"> 提示这个大概是啥毛病呢

哦哦，我仔细看了你的描述，原来是要加[""]😄，谢谢

**Comment by oopuuu** [comment]
现已解决

**Comment by huang770101** [comment]
小爱同学的传感器，事件不能显示吗？
